encoding/protobq/schema.go

```diff
@@ -191,7 +193,10 @@ func (o SchemaOptions) inferFieldSchemaType(field protoreflect.FieldDescriptor)
 		case wkt.BytesValue:
 			return bigquery.BytesFieldType
 		case wkt.Struct:
-			return bigquery.StringFieldType // JSON string
+			if o.UseJSONFieldTypeForWktStruct {
```

Comment:
Can we find a shorter field name? Why not `UseJSONStructs`?

Reply:
This was my suggestion, but I agree that a shorter name would probably be nice.